low-level/sensors/impl/centos_7/drive_manager.py

```diff
@@ -124,13 +130,7 @@ def run(self):
         except Exception as ae:
             # Check for debug mode being activated when it breaks out of blocking loop
             self._read_my_msgQ_noWait()
-            if self.is_running() is True:
-                self._log_debug("DriveManager ungracefully breaking " \
-                                "out of iNotify Loop, restarting: %r" % ae)
-                self._scheduler.enter(1, self._priority, self.run, ())
-            else:
-                self._log_debug("DriveManager gracefully breaking out " \
-                                "of iNotify Loop, not restarting.")
+            raise Exception(f"Failed in monitoring drive events, {ae}")

```

Comment:
Answer in,
https://github.com/Seagate/cortx-monitor/pull/511#discussion_r702836969

Reply:
I dont get it. answer is in old code to know graceful vs ungraceful shutdown. but anyways scheduling back in case ungraceful path is happening now in common recovery mechanism, so its fine.